scripts/rv_expression_association/get_rv_outliers.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+import click
+import logging
+import hail as hl
+import pandas as pd
+from cpg_utils.hail_batch import reference_path
+from cpg_utils.hail_batch import output_path, init_batch
+from cloudpathlib import AnyPath
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+# get OneK1K sample ID and gene name as arguments using click
+@click.command()
+@click.option('--onek1k-id', required=True)
+@click.option('--gene-name', required=True)
+@click.option('--sample-mapping-file', required=True)
+@click.option('--gene-file', required=True)
+@click.option(
+    '--window-size',
+    required=True,
+    help='the size of the flanking region, to be added twice, on either side of the gene body',
+)
+def main(
+    onek1k_id: str,
+    gene_name: str,
+    sample_mapping_file: str,  # 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv'
```

Comment:
Can you map this through `dataset_path` instead, so you don't need to adjust the parameters when running in different access levels?

Reply:
you mean instead of passing it as an argument? Or in the script, instead of (or in addition of) AnyPath??

Reply:
ok, i've done the former, assuming that i still also need AnyPath for pandas to find it?